tests/kuery/test_eql2kql.py

```diff
@@ -51,3 +52,10 @@ def test_ip_checks(self):
         self.validate("dest:192.168.255.255", "dest == '192.168.255.255'")
         self.validate("dest:192.168.0.0/16", "cidrMatch(dest, '192.168.0.0/16')")
         self.validate("dest:192.168.0.0/16", "cidrMatch(dest, '192.168.0.0/16')")
+
+    def test_wildcard_field(self):
+        self.validate('field:"value-*"', 'field == "value-*"')
```

Comment:
all of these values should be unquoted in KQL. Quoted wildcards are treated as string literals. So if the tests fail, the fix is incomplete
```suggestion
        self.validate('field:value-*', 'field == "value-*"')
```

Reply:
I think I misinterpreted the original issue backwards. I'll relook at this.